packages/amplication-client/src/Entity/EntityList.tsx

```diff
@@ -82,7 +82,7 @@ const EntityList: React.FC<Props> = ({ match, innerRoutes }) => {
   const isUserEntityMandatory =
     pluginInstallations?.filter(
       (x) =>
-        x.configurations?.requireAuthenticationEntity === "true" && x.enabled
+        x?.configurations?.requireAuthenticationEntity === "true" && x.enabled
```

Comment:
what will happen if the first equation is undefined or null? I mean this part: `x?.configurations?.requireAuthenticationEntity`
I think the `.length` will throw

Reply:
Yeah, missed that would be adding it. Thanks 🙇🏻

Reply:
Added that @abrl91.